lib/classroom/scopes.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 module Classroom
   module Scopes
-    TEACHER                  = %w(user:email repo delete_repo admin:org).freeze
-    GROUP_ASSIGNMENT_STUDENT = %w(admin:org user:email).freeze
-    ASSIGNMENT_STUDENT       = %w(user:email).freeze
+    TEACHER                  = %w(user:email repo delete_repo admin:org admin:org_hook).freeze
```

Comment:
@cyhsutw I would just make a new constant, and then when we ask for the scope in the controller check for the `current_user` then.